fix(select): make floating labels work for ion-select

imgx64 opened the pull request:
#### Short description of what this resolves:
This makes floating labels work with ion-select, instead of behaving like stacked labels.

#### Changes proposed in this pull request:
- fix(select): make floating labels work for ion-select

**Ionic Version**: 2.x

**Fixes**: #10751

manucorporat commented on src/components/label/label.scss:
```diff
@@ -65,7 +65,3 @@ ion-label[floating] {
 
   max-width: 100%;
 }
-
```
why is this removed?

imgx64 replied:
This CSS rule was forcing the floating label to always "float". It was needed because floating labels only float when the ion-item has `input-has-value` class. Now that it's being set by the ion-select, there's no need to force it.

manucorporat replied:
Fair enough

imgx64 commented:
Once I fixed `input-has-value`, I realized that it made it possible to make floating labels work just by deleting a single CSS rule. So I did just that, and changed the purpose of the change.

I think that CSS rule was added as a workaround because ion-select didn't support `input-has-value`.

manucorporat commented:
Merged! thank you very much for the PR!
